beanstalk_worker/views.py

```diff
@@ -22,3 +25,54 @@ def run_all(request):
     count = task_service.run_all()
 
     return HttpResponse(f"Ran {count} tasks")
+
+
+@csrf_exempt
+def task_launcher(request, task_name, user_name):
+    try:
+        the_user = User.objects.get(username=user_name)
+    except User.DoesNotExist:
+        return JsonResponse({"status": "fail", "error": f"User not found for {user_name}"})
+    except Exception as e:
+        return JsonResponse(
+            {"status": "fail", "error": f"Error while loading user {user_name}", "error_details": str(e)}
+        )
+
```

Comment:
I would add a special check for if the task is in polio plugin and early and quietly return in that case to not spam useless errors (since most env are not polio but all the cron are in polio ).

You can use for this `apps.is_installed("plugins.polio")`  or check the plugin settings, not sure which is better